ext/com_dotnet/com_extension.stub.php

```diff
@@ -54,32 +54,31 @@ function variant_set_type(variant $variant, int $type): void {}
 
 function variant_cast(variant $variant, int $type): variant {}
 
-function com_get_active_object(string $progid, ?int $code_page = null): variant {}
+function com_get_active_object(string $prog_id, ?int $code_page = null): variant {}
```

Comment:
We're using `$code_page` in `sapi_windows_cp_set()`, so we should synchronize them. I prefer writing them separately, but we're using `$codepoint` everywhere, which seems the same grammatical case for me.

Reply:
I'm fine with `$codepage`. :)